pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>servicenow-plugins</artifactId>
-  <version>1.2.0-SNAPSHOT</version>
+  <version>1.2.3-SNAPSHOT</version>
```

Comment:
Keep this version one ahead of whats already there for source plugin. if that is 1.1.0 then make it 1.2.0